packages/paste-core/components/combobox/src/ComboboxListboxOption.tsx

```diff
@@ -6,20 +6,35 @@ import {Text} from '@twilio-paste/text';
 export interface ComboboxListboxOptionProps {
   children: NonNullable<React.ReactNode>;
   highlighted?: boolean;
+  variant: 'default' | 'groupOption';
 }
 
+const getVariantStyles = (variant: ComboboxListboxOptionProps['variant']): {} => {
```

Comment:
I'm not sure which I prefer, but in the Button to Box refactor I started just making a map instead of using switches:

```
const VariantStyles = {
  default: {
    paddingLeft: 'space70',
    paddingRight: 'space70',
  },
  groupOption: {
    paddingLeft: 'space50',
    paddingRight: 'space50',
  }
};

// usage
{...VariantStyles[variant]}
```

Pros: not a function, so no function call. No switch statement verboseness.
Cons: no `default:` case fallback, but with typescript it's unlikely to fall into the default case

Thoughts?

Reply:
Pretty neat and tidy, I like it. Typescript is a bit weird about it, but it's not impossible to type the values so `box` doesn't complain.